vowpalwabbit/cats.cc

```diff
@@ -146,8 +172,17 @@ void reduction_output::print_update_cb_cont(vw& all, const example& ec)
 LEARNER::base_learner* setup(options_i& options, vw& all)
 {
   option_group_definition new_options("Continuous actions tree with smoothing");
-  int num_actions = 0;
-  new_options.add(make_option("cats", num_actions).keep().necessary().help("number of tree labels <k> for cats"));
+  uint32_t num_actions = 0;
+  float bandwidth = 0;
+  float min_value = 0;
+  float max_value = 0;
+  new_options.add(make_option("cats", num_actions).keep().necessary().help("number of tree labels <k> for cats"))
+      .add(make_option("min_value", min_value).keep().help("Minimum continuous value"))
+      .add(make_option("max_value", max_value).keep().help("Maximum continuous value"))
+      .add(make_option("bandwidth", bandwidth)
+               .default_value(1)
+               .keep()
+               .help("Bandwidth (radius) of randomization around discrete actions in terms of continuous range."));
```

Comment:
would it be beneficial to add default value to help message?

Reply:
Actually thanks for pointing it out, should probably change that. `1` is not a good default anymore

Reply:
Changed default in #2788 to be to be half the unit range (i.e. leaf width)